### ISSUE TITLE
fish reads paths.d entries as space-delimited (not line-delimited) values

### ISSUE THREAD

**Issue by dwaite** [fish-shell/fish-shell#5610]
Spotted with VMWare Fusion (local installation version 11.0.1), which adds the file:

```shell
> cat /etc/paths.d/com.vmware.fusion.public 
/Applications/VMware Fusion.app/Contents/Public
```

This will sometimes be added as two entries to PATH.

I believe this is due to line 211 in `config.fish`: `while read -la entry`

Removing the `a` flag fixes this issue for me.

```
fish, version 3.0.0 (installed via home-brew)
uname -a: Darwin Samus-2.local 18.5.0 Darwin Kernel Version 18.5.0: Wed Jan 16 00:04:52 PST 2019; root:xnu-4903.250.305~71/RELEASE_X86_64 x86_64
TERM: xterm-256color
```

**Comment by faho** [comment]
Duplicate of #5481 

which will be fixed in the upcoming 3.0.1.

**Comment by dwaite** [comment]
That was quick. I'm glad I had a chance to learn more about fish trying to track this down :-)
